app/controllers/declaration/AdditionalDocumentChangeController.scala

```diff
@@ -75,6 +75,9 @@ class AdditionalDocumentChangeController @Inject()(
     }
   }
 
+  private def isAuthCodeRequiringAdditionalDocuments(implicit request: JourneyRequest[_]): Boolean =
+    request.cacheModel.isAuthCodeRequiringAdditionalDocuments
+
```

Comment:
Maybe instead of having this method in two classes, you could change the `form` method to accept `ExportsDeclaration`as a parameter? Then you simply write `form(request.cacheModel)`. This is concise and readable and you don't need to create same methods.

Reply:
Also, I think it is better to `ExportsDeclaration` as a parameter, because right now you rely on client of this mehod to know what the `Boolean` param means.